test.rb

```diff
@@ -0,0 +1,45 @@
+require_relative('lens_memprof')
+include LensCMemprof
+puts allocations
+puts "=" * 100
+puts "#{allocations} = 0"
+
+a = 1
+puts "#{allocations} = 0"
+class A
+end
+puts "#{allocations} = 0"
+
+start
+puts "#{allocations} = 0"
+
+puts "-" * 100
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.